src/test/java/seedu/ddd/model/event/DescriptionTest.java

```diff
@@ -7,6 +7,8 @@
 
 import org.junit.jupiter.api.Test;
 
+import seedu.ddd.model.event.common.Description;
```

Comment:
This import should not appear here. Maybe we should to put this DescriptionTest into a new commons folder?